docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+metadata:
+  name: podtato-head
+  namespace: podtato-kubectl
+spec:
+  version: "0.1.0"
+  preDeploymentTasks:
+    - wait-for-prometheus
+  postDeploymentTasks:
+    - post-deployment-loadtests
+  workloads:
+  - name: podtato-head-frontend
+    version: 0.2.0 # Updated version. Needs to be set to the value of the 'app.kubernetes.io/version' label of the updated deployment
+  - name: podtato-head-hat
+    version: 1.1.1
+```
+
+Updating the `KeptnApp` also causes all pre-/post-tasks/evaluations
+of the `KeptnApp` to be executed again.
+In this example, this means that the tasks `wait-for-prometheus`,
+and `post-deployment-loadtests` will run again.
+
+If you are using the [automatic app discovery](../integrate#use-keptn-automatic-app-discovery),
+you do not need to update the `KeptnApp` resource.
+
+After applying the updated manifests, you can monitor the status
+of the application and related workloads using the following commands:
+
+```shell
+$ kubectl get keptnworkloadversion -n podtato-kubectl
+
+NAMESPACE   NAME                                             APPNAME         WORKLOADNAME                         WORKLOADVERSION      PHASE
+podtato-kubectl   podtato-head-podtato-head-frontend-0.1.0   podtato-head    podtato-head-podtato-head-frontend   0.1.0                Completed
+podtato-kubectl   podtato-head-podtato-head-hat-0.1.1        podtato-head    podtato-head-podtato-head-hat        0.1.1                Completed
+podtato-kubectl   podtato-head-podtato-head-frontend-0.2.0   podtato-head    podtato-head-podtato-head-frontend   0.2.0                Completed
+```
+
+As can be seen in the output of the command, the `KeptnWorkloadVersions` from the previous deployment
+are still here, but a new `KeptnWorkloadVersion` for the updated workload has been added.
+For the workload that remained unchanged (`podtato-head-hat`), no new `KeptnWorkloadVersion`
+needed to be created.
+
+Similarly, retrieving the list of `KeptnAppVersions` will reflect the update by
+returning a newly created `KeptnAppVersion`.
+
+```shell
+$ kubectl get keptnappversion -n podtato-kubectl
+
+NAMESPACE         NAME                          APPNAME        VERSION   PHASE
+podtato-kubectl   podtato-head-0.1.0-6bch3iak   podtato-head   0.1.0     Completed
+podtato-kubectl   podtato-head-0.1.0-hf52kauz   podtato-head   0.1.0     Completed
+```
+
+## Adding a new Workload to an Application
+
+To add a new workload (e.g. a new deployment) to an existing app,
+you must:
+
+* Make sure the
+`keptn.sh/app`/`app.kubernetes.io/part-of` label/annotation is present
+on the new workload
+* Add the new workload to the `KeptnApp`,
+if you have previously defined the `KeptnApp resource manually.
+If the application has been discovered automatically, this step is not needed.
+
+For example, to add the deployment `podtato-head-left-leg` to the
+`podtato-head` application, the configuration for that new deployment
+would look like this, with the required label being set:
+
+```yaml
```

Comment:
please use external asset files and include them here for all the code examples in this file :)